src/models/docdbUtils.js

```diff
@@ -1,57 +1,80 @@
+/*
+** DocumentDB Utilities
+**  - Useful, reusable code for our application
+*/
+"use strict";
+
+
 var DocumentDBClient = require('documentdb').DocumentClient;
 
 var DocDBUtils = {
-  getOrCreateDatabase: function(client, databaseId, callback) {
-    var querySpec = {
-      query: 'SELECT * FROM root r WHERE r.id=@id',
-      parameters: [{
-        name: '@id',
-        value: databaseId
-      }]
-    };
-
-    client.queryDatabases(querySpec).toArray(function(err, results) {
-      if (err) {
-        callback(err);
-      }
-
-      if (!err && results.length === 0) {
-        client.createDatabase({
-          id: databaseId
-        }, function(err, created) {
-          callback(null, created);
+    // getOrCreateDatabase
+    //  Connect to existing database or create it (by databaseId)
+    getOrCreateDatabase: function (client, databaseId, callback) {
+        var querySpec = {
```

Comment:
Not part of your change, but queryDatabases should be replaced by readDatabase (as a best practice). It's more lightweight and easier to read